.github/workflows/ci-kpinetwork-frontend.yml

```diff
@@ -96,4 +102,43 @@ jobs:
 
       - name: Terraform Plan Status
         if: steps.plan.outcome == 'failure'
-        run: exit 1
\ No newline at end of file
+        run: exit 1
+
+      - name: Terraform Apply
+        env:
+          TF_VAR_aws_access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          TF_VAR_aws_secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          TF_VAR_aws_account_id: ${{ secrets.AWS_ACCOUNT_ID }}
+          TF_VAR_aws_terraform_user_provider: ${{ secrets.AWS_TERRAFORM_USER_PROVIDER }}
+        run: |
+          cd infrastructure
+          terraform apply -auto-approve
+
+      - name: Get Terraform outputs
+        id: tf-outputs
+        run: |
+          cd infrastructure
+          outputDistributionID=$(terraform output -raw distribution_id)
+          outputRegion=$(terraform output -raw region)
+          outputBucket=$(terraform output -raw bucket_name)
+          echo "::set-output name=distribution_id::$outputDistributionID"
+          echo "::set-output name=region::$outputRegion"
+          echo "::set-output name=bucket_distribution::$outputBucket"
+
+      - name: Sync S3
+        uses: jakejarvis/s3-sync-action@master
+        env:
+          SOURCE_DIR: './dist/apps/cloud'
+          AWS_REGION: ${{ steps.tf-outputs.outputs.region }}
+          AWS_S3_BUCKET: ${{ steps.tf-outputs.outputs.bucket_distribution }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+      - name: Invalidate Cloudfront Distribution
```

Comment:
I'm not sure if we need to deploy every time the CI process is executed, because every time that we push something in a PR to master we'll make a deployment to demo. I think we need to discuss it.